test/VM.js

```diff
@@ -42,16 +51,16 @@ describe("VM", function () {
     ).deploy(supply);
   });
 
-  function execute(commands, state) {
+  function execute(commands, state, overrides) {
     let encodedCommands = commands.map(([target, func, inargs, outargs]) =>
       ethers.utils.concat([
-        target.interface.getSighash(func),
+        func ? target.interface.getSighash(func) : fallBackSelector,
```

Comment:
I see, the `fallBackSelector` is set here.

Reply:
Except I think this ternary operator is saying 
"If func is present, give the sigHash, else give fallback selector"

but I think it should say 
"if func is in the interface, give the sigHash, otherwise give the fallbackSelector"

Reply:
yeah but this is just for tests and i don't want to wrap `target.interface.getSighash(func)` in `try.. catch`